stage1/stdlibdirs.mk

```diff
@@ -8,8 +8,12 @@
 
 ifeq ($(SLD_INCLUDED),)
 
+SLD_SPACE :=
+SLD_SPACE +=
+
 SLD_INCLUDED := x
 SLD_LOCATIONS := $(shell ld --verbose | grep SEARCH_DIR | sed -e 's/SEARCH_DIR("=*\([^"]*\)");*/\1/g')
+SLD_LOCATIONS := $(foreach l,$(SLD_LOCATIONS),$l$(SLD_SPACE)$l/systemd)
```

Comment:
So, after short investigation, we noticed that systemd binaries linking to the libsystemd-shared.so which is not placed in the usual libdir (it is something like `/usr/lib/systemd`) have an RPATH set.

For now I'm fine with hardcoding `/usr/lib/systemd` or else, because `src` flavor is meant to be used by developers only.